src/main/java/net/sf/jabref/autosave/event/AutosaveEvent.java

```diff
@@ -0,0 +1,10 @@
+package net.sf.jabref.autosave.event;
+
+import net.sf.jabref.autosave.Autosaver;
+
+/**
+ * This Event is fired from {@link Autosaver} in case that a save task is pending.
+ */
+public class AutosaveEvent {
```

Comment:
This is just an Object which is passed through the eventBus according to the context. It carries no special data as it's only used for notifying methods which are listening for `AutosaveEvent`. I'm following this strategy to preserve the MVC pattern. Otherwise I could call saving methods directly from `Autosaver` without using the eventBus.